### PR TITLE
Update OVN to 23.09 (latest LTS) and OpenvSwitch to 3.3 (latest LTS) on Rocky Linux 9

### PR DESCRIPTION
**Pull request by mnasiadka**

### CODE REVIEW

**Review by cityofships — DISMISSED** [review]

**Review by cityofships — APPROVED** [review]

File: releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    ``OVN`` version in Rocky Linux 9 container images has been updated to
+    ``23.09``.
```
Comment (markgoddard) [inline releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml:5]: Looks like it affects both RL9 and Ubuntu, and also includes OVS 3.3.

Comment (markgoddard) [reply releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml:5]: @mnasiadka tells me we already had OVS 3.3, this is just a more explicit pin.

Comment (mnasiadka) [reply releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml:5]: Well, I've rebuilt Ubuntu just for freshness, but Ubuntu/Antelope ovs is 3.1.x in UCA and ovn is 22.03 (from Jammy packages, not UCA)

Comment (markgoddard) [reply releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml:5]: Ok, we'll need to consider how this interacts with @seunghun1ee's work to update all packages and container images.

Comment (markgoddard) [reply releasenotes/notes/ovn-23-09-1c710a01b00604b1.yaml:5]: I've just merged @seunghun1ee's PR #1062. The images in this PR will need to be rebuilt using latest stackhpc/2023.1

**Review by markgoddard — CHANGES_REQUESTED** [review]

**Pull request closed without merging**